asset/css/markbind.css

```diff
@@ -297,3 +297,28 @@ panel[minimized]:not([expanded])::before {
 li.footnote-item:target {
     background-color: #eee;
 }
+
+pre {
```

Comment:
Should this rule be changed to apply on `.hljs` instead? I doubt it will have an observable effect but it would be better practice.

Reply:
You mean `pre.hljs` instead?